pybamm/models/submodels/electrolyte_diffusion/composite_diffusion.py

```diff
@@ -20,13 +19,20 @@ class Composite(Full):
     extended : bool
         Whether to include feedback from the first-order terms
 
-    **Extends:** :class:`pybamm.electrolyte.stefan_maxwell.diffusion.Full`
+    **Extends:** :class:`pybamm.electrolyte_diffusion.BaseElectrolyteDiffusion`
     """
 
     def __init__(self, param, reactions, extended=False):
         super().__init__(param, reactions)
         self.extended = extended
 
+    def get_fundamental_variables(self):
+        c_e_n = pybamm.standard_variables.c_e_n
+        c_e_s = pybamm.standard_variables.c_e_s
+        c_e_p = pybamm.standard_variables.c_e_p
+
+        return self._get_standard_concentration_variables(c_e_n, c_e_s, c_e_p)
+
```

Comment:
Quite a few commented lines of code in this file that can probably be removed (not added in this issue).